NautilusCode/program_list.py

```diff
@@ -144,3 +144,7 @@
 progs += Program('zed', _('Zed'),
                  Native('zed', 'zeditor', 'zedit'),
                  Flatpak('dev.zed.Zed'))
+
+progs += Programs('zed-preview', _('Zed Preview'),
+				  Native('zed', 'zeditor', 'zedit'),
```

Comment:
1. Regular "Zed" also has the same Native command line. This will show both "Zed", and "Zed Preview" even when only one of them is installed. We can drop the Native option from "Zed Preview".
2. We use "$name ($build_type)" template for alternative builds of a code editor. So, it should be "Zed (Preview)" with the parentheses.